pom.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>20.2</version>
+    </parent>
+
+    <groupId>io.gravitee.ae</groupId>
+    <artifactId>gravitee-alert-engine-connectors</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Gravitee.io - Alert Engine - Connectors</name>
+
+    <modules>
+        <module>gravitee-alert-engine-connector-api</module>
+        <module>gravitee-alert-engine-connector-core</module>
+        <module>gravitee-alert-engine-connector-ws</module>
+    </modules>
+
+    <properties>
+        <gravitee-bom.version>2.4</gravitee-bom.version>
+        <gravitee-node.version>1.24.0</gravitee-node.version>
+        <gravitee-plugin.version>1.23.1</gravitee-plugin.version>
+        <gravitee-alert-api.version>1.9.0</gravitee-alert-api.version>
+        <mockito.version>4.5.1</mockito.version>
+        <junit-jupiter.version>5.8.2</junit-jupiter.version>
```

Comment:
junit-jupiter is imported in bom 2.5, you don't need to specify its version here